minidump-stackwalk/stackwalker.cc

```diff
@@ -687,6 +735,10 @@ static void ConvertProcessStateToJSON(const ProcessState& process_state,
     }
     thread["frames"] = stack;
     thread["frame_count"] = stack.size();
+    auto thread_name = thread_id_name_map[raw_stack->tid()];
+    if (!thread_name.empty()) {
+      thread["thread_name"] = thread_name;
```

Comment:
Thanks for the suggestion. Though we shouldn't have problems with regex.h functions since it conforms to POSIX.1-2001, POSIX.1-2008, the code is harder to read. The PR is updated with the simpler implementation of split() and stripquotes().